package.json

```diff
@@ -22,6 +22,7 @@
     "@sentry/integrations": "5.6.0-beta.4",
     "@sentry/typescript": "^5.3.0",
     "@types/echarts": "^4.1.10",
+    "@types/classnames": "^2.2.0",
```

Comment:
What do we need the classname types for?

Reply:
since we import them in MenuItem...?